tests/test_akafka.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2021 - 2022 Universität Tübingen, DKFZ and EMBL
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Testing Apache Kafka based providers."""
+
+import json
+import random
+import string
+
+from kafka import KafkaConsumer
+from testcontainers.kafka import KafkaContainer
+
+from hexkit.eventpub.providers.akafka import KafkaEventPublisher
+from hexkit.utils import exec_with_timeout
+
+
+def test_kafka_event_publisher():
+    """Test the KafkaEventPublisher."""
+    random_string = "".join(random.choices(string.ascii_letters, k=10))
```

Comment:
I wanted to make sure that the string that I am producing is fresh and has not been produced by an old test as a side effect. However, I think we can exclude side effects here because we are setting up and tearing down the testcontainer every time we run the test. Will replace with some basic string.

Reply:
Done